src/pages/index.tsx

```diff
@@ -1,7 +1,63 @@
-import React from 'react'
+import { useCallback, useEffect, useState } from 'react'
+import type { DeepReadonly, SelectValue, UseState } from '@okp4/ui'
 import type { NextPage } from 'next'
-import Home from './home/index'
+import type { Dataspace } from '../components/dataspaceDashboard/dataspaceSummary/DataspaceSummary'
+import { DataspaceSummary, DataspaceEntities, DataspaceOptions } from '../components/index'
+import './index.scss'
+import type { Config } from './api/config'
 
-const HomePage: NextPage = () => <Home />
+type HomeProps = {
+  governanceUrl: string
```

Comment:
I think this is not needed anymore as you retrieve the `governanceUrl` through an API call, not through props.

Reply:
of course, I missed it🕵🏻‍♀️